src/constants.rs

```diff
@@ -1,4 +1,5 @@
 pub const FUELUP_PATH: &str = ".fuelup";
+pub const FUELUP_BIN_PATH: &str = ".fuelup/bin";
```

Comment:
I realise we already have a bash script which is unix-specific, but let's not also hard-code unix-style paths if we can avoid it. Instead let's use a `fuelup_bin_dir` function that uses `Path::join` or something like that.